ecl/hthor/hthor.cpp

```diff
@@ -540,6 +549,18 @@ void CHThorDiskWriteActivity::resolve()
             }
 
             clusterHandler.setown(createClusterWriteHandler(agent, NULL, &helper, dfsLogicalName.get(), filename, extend, false));
+            StringBuffer planeName;
+            if (clusterHandler)
+            {
+                StringArray clusterNames;
+                clusterHandler->getClusters(clusterNames);
+                planeName.set(clusterNames.item(0)); // NB: only bother with 1st, if multilple createClusterWriteHandler validates if same
+            }
+            else
+                getDefaultStoragePlane(planeName);
+            bool outputCompressionDefault = agent.queryWorkUnit()->getDebugValueBool("compressAllOutputs", isContainerized());
```

Comment:
Is the workunit option useful?  Not convinced, but may as well leave it in.

Reply:
it isn't useful in cloud, but it would the current way to change the default in bare-metal since you can't configure planes there.